tests/unit/cli/storage_layout/test_storage_layout_overrides.py

```diff
@@ -26,6 +28,56 @@ def test_storage_layout_overrides():
     assert out["layout"] == expected_output
 
 
+def test_storage_layout_overrides_json():
+    code = """
+a: uint256
+b: uint256"""
+
+    storage_layout_overrides = {
+        "a": {"type": "uint256", "slot": 1, "n_slots": 1},
+        "b": {"type": "uint256", "slot": 0, "n_slots": 1},
+    }
+
+    input_json = {
+        "language": "Vyper",
+        "sources": {"contracts/foo.vy": {"content": code}},
+        "storage_layout_overrides": {"contracts/foo.vy": {"content": storage_layout_overrides}},
```

Comment:
i don't think we need to hide it inside the "content" key, since it's already a dict.